pepper-apis/studybuilder-cli/studies/osteo/patches/osteo-version-3-changes.conf

```diff
@@ -0,0 +1,273 @@
+{
+  "study": {
+    "guid": "CMI-OSTEO"
+  },
+  "activity-updates": [
+    {
+      "versionTag": "v3",
+      "activityCode": "PARENTAL_CONSENT",
+      "variable-updates": [
+        {
+          "name": "v2_osteo_consent_minors_s1_participation_detail_p6",
```

Comment:
Do we see any additional advantage by renaming the variable and adding v2_ ?

Reply:
This is what the variable is named currently. Are you saying we should rename the variable to v3_?

Reply:
If thats current name, I am good. No renaming needed . Thanks